pints/tests/test_diagnostics.py

```diff
@@ -79,36 +79,99 @@ def test_within(self):
         # matrix with two columns of samples
         x = np.array([[1.0, 1.1, 1.4, 1.3, 1.3],
                       [1.0, 2.0, 3.0, 4.0, 5.0]])
-        self.assertAlmostEqual(pints._diagnostics.within(x), 1.2635, 5)
+        self.assertAlmostEqual(pints._diagnostics._within(x), 1.2635, 5)
 
     def test_between(self):
         # Tests between chain variance calculation
 
         # matrix with two columns of samples
         x = np.array([[1.0, 1.1, 1.4, 1.3, 1.3],
                       [1.0, 2.0, 3.0, 4.0, 5.0]])
-        self.assertAlmostEqual(pints._diagnostics.between(x), 7.921, 4)
+        self.assertAlmostEqual(pints._diagnostics._between(x), 7.921, 4)
 
-    def test_reorder(self):
-        # Tests that reorder function reshapes correctly
+    def test_rhat(self):
+        # Tests that rhat works
 
-        test = np.random.normal(loc=0, scale=1, size=(4, 10, 3))
-        y = np.array(pints._diagnostics.reorder(0, test))
-        self.assertEqual(y.shape, (8, 5))
+        # Tests Rhat computation for one parameter, chains.shape=(2, 4)
+        chains = np.array([[1.0, 1.1, 1.4, 1.3],
+                          [1.0, 2.0, 3.0, 4.0]])
+        self.assertAlmostEqual(
+            pints._diagnostics.rhat(chains), 2.3303847470550716, 6)
```

Comment:
This looks like a new value of Rhat versus the old test value...

Reply:
This is because the old Rhat did not split the chains in half.

Reply:
Cool